implementations/c/ockam/key_agreement/xx/tests/init_vault.c

```diff
@@ -0,0 +1,71 @@
+#include "init_vault.h"
+
+#include <ockam/vault/default.h>
+#include <ockam/key_agreement.h>
+
+#if OCKAM_ENABLE_ATECC608A_TESTS
+
+#include "ockam/vault/atecc608a.h"
+#include "cryptoauthlib.h"
+#include "atca_cfgs.h"
+#include "atca_iface.h"
+#include "atca_device.h"
+
+ATCAIfaceCfg test_atecc608a_cfg =
+        {
+                .iface_type = ATCA_I2C_IFACE,
+                .devtype = ATECC608A,
+                .atcai2c.slave_address = 0xC0,
```

Comment:
I've created an issue to track the possibility of using more inclusive language for this in the upstream: https://github.com/MicrochipTech/cryptoauthlib/issues/193